website/theme.config.tsx

```diff
@@ -6,6 +6,16 @@ import { Badge } from "./components/Badge";
 import { Callout, useConfig } from "nextra-theme-docs";
 import { components } from "./utils/components";
 import { getSlackAppLink } from "./utils/slack-app-link";
+import { WebSite, WithContext } from "schema-dts";
+import Script from "next/script";
+
+const jsonLd: WithContext<WebSite> = {
+  "@context": "https://schema.org",
+  "@type": "WebSite",
+  name: "CloudQuery",
+  alternateName: ["CloudQuery Docs"],
+  url: "https://docs.cloudquery.io/",
+};
```

Comment:
Is it okay to show this `WebSite` type on every page of the docs? I'm no expert on this but I would have thought that we should use "WebPage" entity on every page

Reply:
> Is it okay to show this `WebSite` type on every page of the docs? I'm no expert on this but I would have thought that we should use "WebPage" entity on every page

Google says you only need to add it to your homepage 
![image](https://github.com/user-attachments/assets/fdefddd5-57da-4cc3-a3f5-e7a6140be994)

We have a redirect from `https://docs.cloudquery.io/` to `https://docs.cloudquery.io/docs` so I'm not sure which one is considered to be the homepage actually

Reply:
This PR adds it to every page but I can make it conditional based on the path

Reply:
> This PR adds it to every page but I can make it conditional based on the path

Yeah, I don't know but if we can do that it seems like it would be a little more in line with what Google suggests

Reply:
@erezrokah @hermanschaaf Using WebPage Schema and IsPartOf seems like a good solution here https://schema.org/WebPage

Reply:
> @erezrokah @hermanschaaf Using WebPage Schema and IsPartOf seems like a good solution here https://schema.org/WebPage

I'm happy to do whatever, what should we put in `IsPartOf`?

Reply:
> Let's use `WebSite` on the home page, and `WebPage` on other pages, with `IsPartOf` set to the home page URL on those other pages? But I think for Google all that really matters is the `WebSite` entry at the root level

This is done (see recent changes). But we have a permanent redirect from the root to `/docs`:
![image](https://github.com/user-attachments/assets/39220a95-a54b-4d2f-b1a0-e8988f496b39)

So not sure if Google reads the root data. The `/docs` path has this now 
![image](https://github.com/user-attachments/assets/46049947-3673-42d1-b7a9-0454b1304f51)